nestkernel/conn_builder.cpp

```diff
@@ -337,6 +337,61 @@ nest::ConnBuilder::single_connect_( index sgid,
   }
 }
 
+inline void
+nest::ConnBuilder::skip_conn_parameter_( thread target_thread, librandom::RngPtr& rng )
+{
+  if ( param_dicts_.empty() ) // indicates we have no synapse params
+  {
+    if ( default_weight_ )
```

Comment:
This is exactly the same code as at the end of the `else` clause. Thus, we can run it unconditionally and turn the test around to `not param_dicts_.empty()` as condition for what currently is the `else` clause.